CHANGELOG.md

```diff
@@ -4,11 +4,57 @@ This document outlines major changes between releases.
 
 ## [Unreleased]
 
+## [0.30.0] - 2024-03-26
+
 ### Added
-- Passing an empty meta parameter value raises "Your metadata headers are not supported." error.
+- Preallocate buffers to object uploads (#835)
+- Check x-amz-expires header on maximum border (#893)
+- Return unimplemented error on GetPublicAccessBlock (#896)
+- Support for aws-chunked (#914)
+- Return unimplemented error in PutBucketLogging (#925)
+- Return not supported error for PutObjectLegalHold command if OFF state (#889)
+- Store CORS to the object (#890)
 
 ### Changed
-- Go 1.20+ is required to build now, using 1.22 by default (#928)
+- Go 1.20+ is required to build now, using 1.22 by default (#928, #933)
+- Limit objects amount for DeleteObjects (#849)
+- Handle bad user id more gracefully (#879)
+- Actualize CODEOWNERS (#942)
+- Passing an empty meta parameter value raises "Your metadata headers are not supported." error (#848)
+- Sdk optimizations (#839)
+- Store bucket owner pub key in container attributes. It helps to process ACL correctly (#915)
+
+### Updated
+- Documentation about ETag (#856)
+- Use SDK with pool base multi sessions (#855)
+- Documentation about Authenticated Users group (#871)
+- Return correct error on lock configuration check (#888)
+- SDK to use slicing optimization (#923)
+- Don't use neofs-crypto directly in neofs/tree (#932)
+- build(deps): bump golang.org/x/net from 0.14.0 to 0.17.0 (#851)
+- build(deps): bump github.com/nats-io/nats-server/v2 from 2.7.4 to 2.9.23 (#867)
+- build(deps): bump google.golang.org/grpc from 1.57.0 to 1.57.1 (#880)
+- build(deps): bump github.com/nats-io/nkeys from 0.4.4 to 0.4.6 (#894)
+- build(deps): bump golang.org/x/crypto from 0.14.0 to 0.17.0 (#917)
+- build(deps): bump google.golang.org/protobuf from 1.31.0 to 1.33.0 (#935)
+
+### Removed
+- Removed 'v' from app version (#922)
+- Removed re-slicing for multipart (#931)
```

Comment:
"Removed removed" twice now?

Reply:
There is no point in writing "removed" in "removed" section. Consider
```
Removed:
 * this
 * that
```
and
```
Removed:
 * removed this
 * removed that
```
Same goes for "fixed fixed"/"added added"/etc.

Reply:
Ow, now I see. Just misunderstood, the first comment was `Removed Remove` for two rows. I decided it was suggestion for row changing